com.stansassets.build/Editor/Steps/IBuildStep.cs

```diff
@@ -2,9 +2,10 @@
 {
     public interface IBuildStep
     {
-        bool Execute(BuildContext buildContext);
+        event Action<ExecuteFinishedArgs> OnExecuteFinished;
+
+        void Execute(BuildContext buildContext);
 
         int Priority { get; }
-        string GetResultMessage();
```

Comment:
Where `ExecuteFinishedArgs` ? 

regarding the API the same comment how do I know as a developer when I need to call `OnExecuteFinished `
 void Execute(BuildContext buildContext, Action<ExecuteFinishedArgs> onComplete);